tests/test_integration.py

```diff
@@ -229,10 +235,11 @@ def test_robot_verbose(self):
             'tests/test_in/robot_double_fail.xml',
         ])
         self.assertEqual(1, retval)
-        self.assertEqual(["Suite One &amp; Suite Two.Suite Two.Another test",
-                          "number of warnings (1) is higher than the maximum limit (0).",
-                          "Returning error code 1."],
-                         self.caplog.messages)
+        self.assertEqual(
+            ["Robot: suite 'Suite Two'    Suite One &amp; Suite Two.Suite Two.Another test",
+             "Robot: suite 'Suite Two'    number of warnings (1) is higher than the maximum limit (0).",
+             'Robot: Returning error code 1.'],
```

Comment:
Please make consequent use of the same quotes in the same list.

Reply:
I generated this with `pprint`. I prefer to keep it as is to keep the diff to a minimum when using `pprint`'s output again in the future, if we decide to reformat our logs again. What do you think?

Reply:
I changed all files in 6fde428cdfca077fabd76737e54b0705b350a6a6
I personally don't like the mix of the 2 quotes (certainly not in the same list).